authorization_code/public/index.html

```diff
@@ -0,0 +1,142 @@
+<!doctype html>
```

Comment:
I'm not going to go through this whole file.

Basically its an html page that uses templates and a bit of javascript.
It looks at the url to pull out spotify access tokens and uses AJAX to make http requests from the browser.